repo/importer.py

```diff
@@ -73,14 +74,22 @@ async def read_and_display(*cmd, cwd=os.getcwd(), stdin=None):
         rc = await process.wait()
     return rc, stdout, stderr
 
+def get_token():
+    token = os.environ.get('GITHUB_TOKEN')
+    if not token:
+        raise Exception("No token in env")
+    
+    return token
 
 def is_valid_repo(repo_name):
-    g = git.cmd.Git()
-    try:
-        g.ls_remote('-h', f'git@github.com:{repo_name}')
-    except:
-        return False
-    return True
+    url = f"https://api.github.com/repos/{repo_name}/events"
+   
+    data = requests.get(
```

Comment:
why the change?

Reply:
This will break the current importer.

Reply:
can get have a different method?

Reply:
Previous version of «is_valid_repo» implied ssh connection, which docker did not have. Current version uses token as env variable. Others functions in importer do not work though. Should we work with both functions?